src/init.cpp

```diff
@@ -673,6 +690,15 @@ bool AppInit2(boost::thread_group& threadGroup)
         nScriptCheckThreads = MAX_SCRIPTCHECK_THREADS;
 
     fServer = GetBoolArg("-server", false);
+    nPrune = GetArg("-prune", 0) * 1024 * 1024;
+    if (nPrune) {
+        if (nPrune >= MIN_BLOCK_FILES_SIZE)
+            LogPrintf("Autoprune configured to use less than %uMiB on disk for block files.\n", nPrune / 1024 / 1024);
+        else {
+            nPrune = ~0;
+            LogPrintf("Autoprune configured below the minimum of %uMiB. Setting at the maximum possible of %uMiB, to avoid pruning too much. Please, check your configuration.\n", MIN_BLOCK_FILES_SIZE / 1024 / 1024, nPrune / 1024 / 1024);
```

Comment:
I think it's more clear if you just say something like "Leaving pruned mode enabled, but not deleting any more blocks for now".